### PR TITLE
[BUG] wrong translation of title and description xliff-Export -- fixed

### PR DESCRIPTION
**Pull request by sara-luise**
## Changes in this pull request  
Resolves ##13141

### CODE REVIEW

**Comment by dvesh3** [comment]
@sara-luise could you please sign the CLA? thanks!

**Comment by sara-luise** [comment]
> sign the CLA? thanks!
I already did, but it seems as it takes some time.

**Comment by jdreesen** [comment]
No, as @CLAassistant says:
> Sara Rittberger seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

File: lib/Translation/ExportDataExtractorService/DataExtractor/DocumentDataExtractor.php [hunk 1]
```diff
@@ -145,8 +145,8 @@ protected function addSettings(Document $document, AttributeSet $result): Docume
                     $targetDocument = Document::getById($translations[$targetLanguage]);
 
                     if ($targetDocument instanceof  Document\PageSnippet) {
-                        $targetData['title'][$targetLanguage] = $document->getTitle();
-                        $targetData['description'][$targetLanguage] = $document->getDescription();
+                        $targetData['title'][$targetLanguage] = $targetDocument->getTitle();
```
Comment (dvesh3) [inline lib/Translation/ExportDataExtractorService/DataExtractor/DocumentDataExtractor.php:148]: please change the if check to `if ($targetDocument instanceof Document\Page) {` to fix the failing tests. thanks!

**Comment by dvesh3** [comment]
@sara-luise thanks!

**Pull request merged**